packages/adapter-squid/src/SquidAdapter.ts

```diff
@@ -59,9 +58,7 @@ export class SquidAdapter
       );
 
       if (!adapterConfigItem) {
-         throw new Error(
-            `${ERR_ADAPTER_INIT_FAILURE}: ${ERR_ADAPTER_CONFIG_MISSING}`
-         );
+         throw new Error(`${ERR_ADAPTER_CONFIG_MISSING}`);
```

Comment:
nit: no need for the template literals here as `ERR_ADAPTER_CONFIG_MISSING` is a string